src/main/scala/com/advancedtelematic/director/db/DeviceUpdate.scala

```diff
@@ -13,16 +13,18 @@ object DeviceUpdateResult {
   sealed abstract class DeviceUpdateResult
 
   final case object NoUpdate extends DeviceUpdateResult
-  final case class UpdateNotCompleted(updateTarget: DeviceUpdateTarget) extends DeviceUpdateResult
-  final case class UpdateSuccessful(updateTarget: DeviceUpdateTarget) extends DeviceUpdateResult
+  final case class UpdateNotCompleted(updateTarget: DeviceUpdateAssignment) extends DeviceUpdateResult
```

Comment:
Maybe we could also change the field name to `updateAssignment`? Or will that propagate to other services and get messy?